### ISSUE TITLE
Problem about kitti2015 eval

### ISSUE THREAD

**Issue by Zhuhan-1997** [mileyan/AnyNet#27]
I downloaded your pre-trained model and used the following command to test.
python finetune.py --maxdisp 192 --datapath path-to-kitti2015/training/ \
    --save_path results/kitti2015 --datatype 2015 --pretrained checkpoint/kitti2015_ck/checkpoint.tar \
    --split_file checkpoint/kitti2015_ck/split.txt --evaluate
without spn.The result is :Average test 3-Pixel Error = Stage 0=0.1401, Stage 1=0.1580, Stage 2=0.1008
It is very different from the best result, which makes me very confused.  
My CUDA 10.1 python 3.7.3  pytorch 1.5.1

**Comment by ZakariyaTaha** [comment]
Hi ! Were you able to solve this problem ?

**Comment by friendship1** [comment]
I am having the same problem with same metric value.  Is there still no answer to this? @mileyan

**Comment by Miragecore** [comment]
set the align_corners=True to nn.functional.upsample & grid_upsample.
I got this.

Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615
torch Version: 1.10.0+cu111
cuda_11.1.TC455_06.29190527_0
Python 3.7.13

**Comment by flyxuexi** [comment]
> set the align_corners=True to nn.functional.upsample & grid_upsample. I got this.
> 
> Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615 torch Version: 1.10.0+cu111 cuda_11.1.TC455_06.29190527_0 Python 3.7.13

could you tell me where to set  the align_corners=True please?,I don't find it in this project

**Comment by flyxuexi** [comment]
> set the align_corners=True to nn.functional.upsample & grid_upsample. I got this.
> 
> Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615 torch Version: 1.10.0+cu111 cuda_11.1.TC455_06.29190527_0 Python 3.7.13

I can't find grid_upsample,I can only find nn.functional.upsample&F.upsample

**Comment by flyxuexi** [comment]
> set the align_corners=True to nn.functional.upsample & grid_upsample. I got this.
> 
> Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615 torch Version: 1.10.0+cu111 cuda_11.1.TC455_06.29190527_0 Python 3.7.13

My environment is the same as yours，
I wonder when ‘cd models/spn_t1 bash make.sh’ ，will this error occur？：
cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
warning: ‘T* at::Tensor::data() const [with T = float]’ is deprecated: Tensor.data<T>() is deprecated. Please use Tensor.data_ptr<T>() instead. [-Wdeprecated-declarations]

**Comment by flyxuexi** [comment]
> set the align_corners=True to nn.functional.upsample & grid_upsample. I got this.
> 
> Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615 torch Version: 1.10.0+cu111 cuda_11.1.TC455_06.29190527_0 Python 3.7.13

May I ask whether you finetune the model trained by yourself or the model given by the author?

**Comment by flyxuexi** [comment]
> set the align_corners=True to nn.functional.upsample & grid_upsample. I got this.
> 
> Average test 3-Pixel Error = Stage 0=0.1999, Stage 1=0.1046, Stage 2=0.0671, Stage 3=0.0615 torch Version: 1.10.0+cu111 cuda_11.1.TC455_06.29190527_0 Python 3.7.13

Soory I can't find grid_upsample,i can only find nn.functional.upsample & F.upsample in anynet.py
After adding align_corners=True,my result become better but still lower than yours:
Average test 3-Pixel Error = Stage 0=0.1382, Stage 1=0.1248, Stage 2=0.0874, Stage 3=0.0798
